### PR TITLE
fix: Added 'A' protomResponse status to STATE_ALARM_ARMED_CUSTOM_BYPA…

### PR DESCRIPTION
**Pull request by bvis**
…SS which represents ARMED_AWAY+PERI

### CODE REVIEW

File: custom_components/securitas/alarm_control_panel.py [hunk 1]
```diff
@@ -63,15 +63,6 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
             )
     add_entities(alarms)
 
-
-def set_arm_state(state, code=None):
```
Comment (bvis) [inline custom_components/securitas/alarm_control_panel.py:67]: It seems this function is not used anywhere, the implementation in the class is used instead

File: custom_components/securitas/alarm_control_panel.py [hunk 2]
```diff
@@ -226,9 +217,10 @@ def update_status_alarm(self, status: CheckAlarmStatus = None):
             elif status.protomResponse == "P":
                 self._state = STATE_ALARM_ARMED_HOME
             elif (
-                status.protomResponse == "E"
-                or status.protomResponse == "B"
-                or status.protomResponse == "C"
+                status.protomResponse == "E" # PERI
+                or status.protomResponse == "B" # PERI + ARMED_HOME
+                or status.protomResponse == "C" # PERI + ARMED_NIGHT
+                or status.protomResponse == "A" # PERI + ARMED_AWAY
```
Comment (bvis) [inline custom_components/securitas/alarm_control_panel.py:223]: Adding this new `protomResponse` and a text explanation of what each means.

File: custom_components/securitas/alarm_control_panel.py (hunk 1 above)
Comment (guerrerotook) [reply custom_components/securitas/alarm_control_panel.py:67]: Yes, you are right

File: custom_components/securitas/alarm_control_panel.py (hunk 2 above)
Comment (guerrerotook) [reply custom_components/securitas/alarm_control_panel.py:223]: Fantastic!

**Pull request merged**